fastlane/Fastfile

```diff
@@ -1,49 +1,107 @@
-# This file contains the fastlane.tools configuration
-# You can find the documentation at https://docs.fastlane.tools
-#
-# For a list of all available actions, check out
-#
-#     https://docs.fastlane.tools/actions
-#
-# For a list of all available plugins, check out
-#
-#     https://docs.fastlane.tools/plugins/available-plugins
-#
-
-# Uncomment the line if you want fastlane to automatically update itself
-# update_fastlane
+require_relative './amplify_pods.rb'
 
 opt_out_usage
-
 default_platform(:ios)
 
-COCOAPODS_PUSH_WAIT_TIME = 600
+PLIST_KEY = "CFBundleShortVersionString"
 
-pods = [
-  {spec: "Amplify.podspec", sleep: true},
-  # AmplifyPlugins depends on AWSPluginsCore, but skipping the sleep here
-  # since there's one after CoreMLPredictionsPlugin
-  {spec: "AWSPluginsCore.podspec", sleep: false},
-  {spec: "CoreMLPredictionsPlugin.podspec", sleep: true},
-  {spec: "AWSPredictionsPlugin.podspec", sleep: false},
-  {spec: "AmplifyPlugins.podspec", sleep: false}
-]
+COCOAPODS_PUSH_WAIT_TIME = 600
 
 platform :ios do
+
+  desc "Create a pre-release version by pushing a tag to GitHub, and pushing pods to CocoaPods"
+  lane :unstable do 
+
+    next_version = calculate_next_canary_version
+
+    UI.message("Release unstable version: #{next_version}")
```

Comment:
Should this be "Releasing.." -- trying to think of what I'd expect to see as a reader of the logs.